src/ducks/apps/selectors.js

```diff
@@ -2,6 +2,8 @@ import keyBy from 'lodash/keyBy'
 import { createSelector } from 'reselect'
 import { queryDataSelector } from 'selectors'
 
+export const getAppURL = app => (app ? app.links.related : null)
```

Comment:
We should be more safe with `app => (app && app.links ? app.links.related : null)`

Reply:
Done thanks